packages/component/.eslintrc.yml

```diff
@@ -10,6 +10,7 @@ rules:
   react/button-has-type: error
   react/default-props-match-prop-types: error
   react/destructuring-assignment: error
+  react/display-name: off
```

Comment:
I think we want this rule. Can you file a bug so we turn it back on later?

Reply:
https://github.com/microsoft/BotFramework-WebChat/issues/3449